views/js/runner/proxy/cache/proxy.js

```diff
@@ -451,6 +481,7 @@ define([
          *                      Any error will be provided if rejected.
          */
         callItemAction: function callItemAction(itemIdentifier, action, params) {
+            console.log('call ItemAction', action);
```

Comment:
Is this needed? Maybe a forgotten debug code?

Reply:
my bad